src/web/components/Cards/Card/components/ImageTopLayout.tsx

```diff
@@ -3,19 +3,14 @@ import { css } from 'emotion';
 
 type Props = {
     children: JSX.Element | JSX.Element[];
-    coverage?: CardCoverageType;
 };
 
-export const ContentWrapper = ({ children, coverage }: Props) => (
+export const ImageTopLayout = ({ children }: Props) => (
```

Comment:
So this card is all about refactoring these layouts to be simpler.

https://trello.com/c/4zGokZSv/887-refactor-card-layout

I've not done anything around responsive layouts yet but hopefully with this refactor it'll be easier. This could well be a fun problem though 🤓